babel/index.babel

```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
```

Comment:
- [ ]  Please make sure you have documented your repo, add a professional readme file

Reply:
Good job asking for changes politely, kindly you can do it like that
_The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
you can use this [template](https://github.com/microverseinc/readme-template)._

Reply:
Good job asking for changes politely, kindly you can do it like that
_The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
you can use this [template](https://github.com/microverseinc/readme-template)._

Reply:
Good job asking for changes politely, kindly you can do it like that
_The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?
you can use this [template](https://github.com/microverseinc/readme-template)._